### DISCUSSION TITLE
Fastlane Getting Freeze at Running script '[CP] Copy Pods Resources' and 'Run Script'

### DISCUSSION THREAD

**Discussion by ChawkiOS** [fastlane/fastlane#18501] [Q&A] [answered]
Hello 👋

I have a problem with the github Actions CI of the iOS application (react native), I have all the lanes that work well, except the build of the application that works well locally and on Travis, but on Github Actions it remains blocked at the

```
Running script '[CP] Copy Pods Resources'
Running script 'Run Script'
```

I removed Running script 'Run Script', and it gets stuck on the step just before!

I have tried a lot of build methods and still don't work!

![lSEih](https://user-images.githubusercontent.com/9388387/113404940-4b7a6280-93a9-11eb-8ded-9d7b363cbeb9.png)

https://stackoverflow.com/questions/66706061/ios-build-not-working-on-fastlane-and-github-action

<details><summary>🚫 fastlane environment 🚫</summary>

### Stack

| Key                         | Value                                                          |
| --------------------------- | -------------------------------------------------------------- |
| OS                          | 11.2.3                                                         |
| Ruby                        | 2.6.3                                                          |
| Bundler?                    | false                                                          |
| Git                         | git version 2.22.0                                             |
| Installation Source         | /usr/local/bin/fastlane                                        |
| Host                        | macOS 11.2.3 (20D91)                                           |
| Ruby Lib Dir                | /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib |
| OpenSSL Version             | LibreSSL 2.8.3                                                 |
| Is contained                | false                                                          |
| Is homebrew                 | false                                                          |
| Is installed via Fabric.app | false                                                          |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/                    |
| Xcode Version               | 12.4                                                           |

**Comment by ChawkiOS** [comment, marked as answer]
Here's the solution I found that works pretty well for me ! Before calling `build_app` must call `setup_ci(force: true)` in Fastfile file.

```
  lane :prepar_archive do |values|
    setup_ci(force: true)
    UI.important "Environment [APP_STORE_CONNECT_API_KEY_KEY]= #{ENV['APP_STORE_CONNECT_API_KEY_KEY']}"
    match(readonly: true)
    buildNumber = ENV["GITHUB_RUN_NUMBER"]
    increment_build_number(xcodeproj: "appneme.xcodeproj", build_number:buildNumber)
    increment_version_number(
      version_number: "1.0."+buildNumber
    )
    gym
  end
```

**Reply by ElectricCookie** [reply]
Thanks!!

**Reply by ragul-pr** [reply]
I have added this still it is stucked
